specutils/manipulation/extract_spectral_region.py

```diff
@@ -99,13 +99,16 @@ def extract_region(spectrum, region):
 
     Parameters
     ----------
-    spectrum: `~specutils.spectra.spectrum1d.Spectrum1D`
+    spectrum: `~specutils.Spectrum1D`
         The spectrum object from which the region will be extracted.
 
+    region: `~specutils.SpectralRegion`
+        The spectral region to extract from the original spectrum.
+
     Returns
     -------
-    spectrum: `~specutils.spectra.spectrum1d.Spectrum1D`
-        Excised spectrum.
+    spectrum: `~specutils.Spectrum1D` or list of `~specutils.Spectrum1D`
+        Excised spectrum, or list of spectra if the input region contained multiple subregions.
```

Comment:
```suggestion
    spectrum: `~specutils.Spectrum1D` or list of `~specutils.Spectrum1D`
        Excised spectrum, or list of spectra if the input region contained multiple 
        subregions.
```
Exceedingly minor pep8 item

Reply:
Having just looked at the details of pep8 in light of #621, they mandate to keep the 72-char limit for docstrings even where longer code lines are allowed, right? flake8 by default seems to assume the same for code and docs.